src/ccl/charts/line-graph.component.ts

```diff
@@ -0,0 +1,119 @@
+import { Component, ViewEncapsulation, ElementRef } from '@angular/core';
+import * as D3 from 'd3';
+
+/*
+ * Line graph component
+ * Contains all logic for drawing a line graph
+ */
+
+@Component({
+  selector: 'line-graph',
+  encapsulation: ViewEncapsulation.None,
+  template: `<ng-content></ng-content>`,
+  inputs: [ 'data', 'indicator' ],
+  styleUrls: [ '../../assets/css/chart.css']
+})
+
+export class LineGraph {
+
+  public data: Object;
```

Comment:
I'm not too up with typescript yet. Can type declaration, as in this case, basically be a comment? i.e. Object<ChartData>
As is there is no appropriate existing class